Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.0)
+    rspec-mocks (3.9.1)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-support (3.9.0)
-    sequel (5.27.0)
+    rspec-support (3.9.3)
+    ruby2_keywords (0.0.2)
+    sequel (5.33.0)
     shotgun (0.9.2)
       rack (>= 1.0)
-    sinatra (2.0.7)
+    sinatra (2.0.8.1)
```

Comment:
### [sinatra](http://sinatrarb.com/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.7 → 2.0.8.1

[_[change-log](https://github.com/sinatra/sinatra/blob/master/CHANGELOG.md), [source-code](https://github.com/sinatra/sinatra)_]





<details>
<summary>Commits</summary>

A change of **32** commits. See the full changes on [the compare page](https://github.com/sinatra/sinatra/compare/v2.0.7...v2.0.8.1).

These are the first 10 commits:
- (4e71133) [readme translation updates](https://github.com/sinatra/sinatra/commit/4e71133aba95c07282f30756ffde23114a093f57)
- (909551e) [Lookup Tilt class for template engine without loading files](https://github.com/sinatra/sinatra/commit/909551e4bb046b0779e075dc1a04f0c1fe54a5f3)
- (71dea98) [CI: Use ruby 2.4.7, 2.5.6, 2.6.4](https://github.com/sinatra/sinatra/commit/71dea98b4437e027ff07b96e11a35d6d4d03369d)
- (243dca5) [Add request method and path to the NotFound exception messag…](https://github.com/sinatra/sinatra/commit/243dca5c40650269cec922ed8decfbf6f5eea83a)
- (1199d64) [Add YAML extension to Sinatra::Contrib::ConfigFile](https://github.com/sinatra/sinatra/commit/1199d643ef99e15e387707c267a41ed469176a55)
- (722353e) [Merge pull request #1564 from EmersonManabuAraki/fix/allow-y…](https://github.com/sinatra/sinatra/commit/722353ee7c02981050ff2658677e8a51a6ddab30)
- (6ee7fc1) [Merge pull request #1565 from olleolleolle/patch-4](https://github.com/sinatra/sinatra/commit/6ee7fc1a42e0e930eb603b9a4b67a4ff7406bfe0)
- (8b76b4e) [CI: Use ruby 2.4.9, 2.5.7, 2.6.5](https://github.com/sinatra/sinatra/commit/8b76b4e346969bccb2859cf778acd925a97e3064)
- (192bc9b) [Merge pull request #1540 from jeduardo824/1249-update-README…](https://github.com/sinatra/sinatra/commit/192bc9b94308fcfaa76a4e9b71daec708aa7b5b4)
- (77b707d) [Add test where pre filter cannot modify params on certain ca…](https://github.com/sinatra/sinatra/commit/77b707daab84801b3ea54748a50a56b5e90a00e8)


</details>